code/components/rage-formats-x/include/convert/phBound_five_rdr3.h

```diff
@@ -213,9 +213,200 @@ static inline void fillPolyhedronBound(rdr3::phBoundPolyhedron* out, five::phBou
 	}
 }
 
-static inline uint8_t ConvertMaterialIndexrdr3(uint8_t index)
+static inline uint16_t ConvertMaterialIndexrdr3(uint8_t index)
 {
-	return index;
+	static std::map<uint8_t, uint16_t> conversionMap;
+
+	#pragma region material conversion mappings
+	if (conversionMap.empty())
+	{
+		conversionMap[0] = 12667; // DEFAULT
```

Comment:
Why hardcoded indexes when you can generate those from material name? :(
Especially given that you left material names as comments.

Reply:
It's not faster than doing this every time when conversion function it's called?

Reply:
Ahh i see how to do this (:

Reply:
`conversionMap` is static so it would be filled only once.